unattended-upgrade

```diff
@@ -863,20 +863,22 @@ def is_pkg_change_allowed(pkg, blacklist, whitelist):
     return True
 
 
-def transitive_dependencies(pkg, cache, acc=set()):
-    # type (apt.Package, apt.Cache, AbstractSet[str]) -> bool
+def transitive_dependencies(pkg, cache, acc=set(), valid_types=None):
+    # type (apt.Package, apt.Cache, AbstractSet[str], AbstractSet[str]) -> bool
     """ All (transitive) dependencies of the package
     """
     # Note that alternative (|) dependencies are collected, too
     valid_types = {'Depends', 'PreDepends', 'Recommends'}
```

Comment:
AFAICT; This is still overriding the parameter.

Reply:
Thanks, fixed that!